### PR TITLE
Keyword `contentMediaType`

### PR DESCRIPTION
**Pull request by AgniveshChaubey**
This PR addresses #81.

### CODE REVIEW

File: content/2020-12/content/contentMediaType.markdown [hunk 1]
```diff
@@ -18,3 +18,55 @@ Annotations
...
 
 This keyword produces the content media type as the annotation value.
+
+## Explanation
+
+The `contentMediaType` keyword in JSON Schema specifies the MIME type of the contents of a string. It is used to annotate the type of media contained within a string. It should be noted that:
+
+* This keyword is purely an annotation and does not directly affect validation.
+* It describes the media type of the binary string after it has been decoded as specified in `contentEncoding`.
+* Using `contentMediaType` without `contentEncoding` may lead to ambiguity or errors.
```
Comment (jviotti) [inline content/2020-12/content/contentMediaType.markdown:28]: Maybe just say that it is recommended to set `contentEncoding` if `contentMediaType` is declared?

File: content/2020-12/content/contentMediaType.markdown [hunk 2]
```diff
@@ -18,3 +18,55 @@ Annotations
...
+* The value assigned to `contentMediaType` must be a valid IANA media type as described in [RFC 2046](https://www.rfc-editor.org/rfc/rfc2046.html).
+* It is recommended to use officially registered IANA media types to ensure interoperability and compatibility.
```
Comment (jviotti) [inline content/2020-12/content/contentMediaType.markdown:30]: Isn't this the same as the point right above?

File: content/2020-12/content/contentMediaType.markdown [hunk 3]
```diff
@@ -18,3 +18,55 @@ Annotations
...
+  "contentEncoding": "base64",
+  "contentMediaType": "application/json"
+}
+{{</schema>}}
+
+{{<instance-pass `An instance with a properly stringified JSON document encoded in base64 is valid`>}}
+"eyAibmFtZSI6ICJKb2huIERvZSIgfQ=="    // --> { "name": "John Doe" }
+{{</instance-pass>}}
+
+{{<instance-pass `A validly-encoded invalid JSON document is also valid`>}}
```
Comment (jviotti) [inline content/2020-12/content/contentMediaType.markdown:47]: "validly-encoded" reads weird. Maybe say "An encoded value that represents invalid JSON data is still valid"?

File: content/2020-12/content/contentMediaType.markdown [hunk 4]
```diff
@@ -18,3 +18,55 @@ Annotations
...
+"eyAibmFtZSI6IH0="    // --> { "name": }
+{{</instance-pass>}}
+
+{{<instance-pass `An instance with non string value is ignored`>}}
```
Comment (jviotti) [inline content/2020-12/content/contentMediaType.markdown:51]:
```suggestion
{{<instance-pass `A non-string instance is ignored`>}}
```

File: content/2020-12/content/contentMediaType.markdown (hunk 2 above)
Comment (AgniveshChaubey) [reply content/2020-12/content/contentMediaType.markdown:30]: Yes, removed it!

**Comment by AgniveshChaubey** [comment]
Updated!

**Review by jviotti — APPROVED** [review]

**Pull request merged**